### DISCUSSION TITLE
Mitigations against “matching public data” actions

### DISCUSSION THREAD

**Discussion by Bl5ckj5ck** [bisq-network/bisq#5299] [General]
Hey guys, I was exploring the api public webpage (api/trades) when I noticed that it shows some info (timestamp, amount, payment-method) that could compromise the privacy of traders. These info though are currently available not just through the api.

Couldn’t this lead  institutions or bad actors to easily find out if a user used Bisq by matching the timestamp, the amount and the payment-method of the tx on the bank side with the info found publicly like in the api?

I know that there are currently few mitigations preventing the identification of a trader: amounts are rounded to full units and the trade doesn’t necessarily close on the date of the fiat payment.
However, I feel like these solutions are not enough to preserve the privacy of the users. 
Wouldn’t be possible to hide some data that currently get published (like the “payment method”) when the trade is executed?
I opened this discussion in order to find ways which could prevent external actors from identifying traders

**Comment by RififiCastorjunior** [comment]
Hi! i can't verrify for the moment, but is that possible to fetch those info outside of a trade, withoud being a pair in the trade ?

**Reply by Bl5ckj5ck** [reply]
Hi! Yes you can view those info just by doing an api request: https://bisq.markets/api/trades?market=btc_eur&limit=2000
